docs/Help.md

```diff
@@ -94,12 +94,12 @@ Nightly is a "developer release" of Chatterino. It is released every time there'
     - For **Linux** download `test-cmake-Chatterino-x86_64.AppImage`
     - For **Mac** download `test-cmake-chatterino-osx.dmg`
 1. Install nightly:
-    - On **Windows**, right click the archive > `Extract All` > `Extract` (Override files if prompted). Open the newly extracted folder and create a shortcut for the `chatterino.exe` file to the Desktop for easy access.
+    - On **Windows**, right-click the archive > `Extract All` > `Extract` (Override files if prompted). Open the newly extracted folder and create a shortcut for the `chatterino.exe` file to the Desktop for easy access.
     - On **Linux**, open up the download directory in your terminal and execute the following command `chmod +x test-cmake-Chatterino-x86_64.AppImage && sudo mv test-cmake-Chatterino-x86_64.AppImage /usr/local/bin`
     - On **Mac**, open the dmg file and drag Chatterino into the Applications folder.
 
 ##### Updating
-Nightly ignores version checking and it will not attempt to update on its own. Because of that it will not notify you when a new stable update is available. To update, repeat steps above. If you want to switch back to stable release, download and install it from [Chatterino's website](https://chatterino.com/#downloads).
+Nightly ignores version checking, and it will not attempt to update on its own. Because of that it will not notify you when a new stable update is available. To update, repeat steps above. If you want to switch back to stable release, download and install it from [Chatterino's website](https://chatterino.com/#downloads).
```

Comment:
Same scenario here, there is no half-pause required.

```suggestion
Nightly ignores version checking and it will not attempt to update on its own. Because of that it will not notify you when a new stable update is available. To update, repeat steps above. If you want to switch back to stable release, download and install it from [Chatterino's website](https://chatterino.com/#downloads).
```